script.js

```diff
@@ -0,0 +1,14 @@
+
+
+
+function webCalculator(){
+    var firstNum, secondNum, sum;
```

Comment:
Don't declare variables in a group like this. You may see people doing it online but it is a code smell (bad habit) we don't want you picking up.